db/data.py

```diff
@@ -150,7 +151,7 @@ def insert_version(connection, data, version_type):
     row = result.fetchone()
     return row[0]
 
-def write_low_level(mbid, data):
+def write_low_level(mbid, data, is_mbid):
```

Comment:
Check the code I already changed this, but I don't know why github does not updated this!